nats.go

```diff
@@ -2507,6 +2507,19 @@ func (nc *Conn) IsConnected() bool {
 	return nc.isConnected()
 }
 
+// Servers returns the list of known servers, including additional
```

Comment:
Well, not really, we return all in srvPool, right? So there can be the one that the user added through Options.Servers